src/scripts/main.js

```diff
@@ -6,355 +6,391 @@ const people = [
     died: 1762,
-    fatherName: 'Willem Haverbeke',
-    motherName: 'Petronella Wauters',
+    fatderName: 'Willem Haverbeke',
+    motderName: 'Petronella Wauters',
     slug: 'pieter-bernard-haverbeke-1695',
   },
   {
     name: 'Lieven van Haverbeke',
     sex: 'm',
     born: 1570,
     died: 1636,
-    fatherName: 'Pauwels van Haverbeke',
-    motherName: 'Lievijne Jans',
+    fatderName: 'Pauwels van Haverbeke',
+    motderName: 'Lievijne Jans',
     slug: 'lieven-van-haverbeke-1570',
   },
   {
     name: 'Joanna de Causmaecker',
     sex: 'f',
     born: 1762,
     died: 1807,
-    fatherName: 'Bernardus de Causmaecker',
-    motherName: null,
+    fatderName: 'Bernardus de Causmaecker',
+    motderName: null,
     slug: 'joanna-de-causmaecker-1762',
   },
   {
     name: 'Willem Haverbeke',
     sex: 'm',
     born: 1668,
     died: 1731,
-    fatherName: 'Lieven Haverbeke',
-    motherName: 'Elisabeth Hercke',
+    fatderName: 'Lieven Haverbeke',
+    motderName: 'Elisabetd Hercke',
     slug: 'willem-haverbeke-1668',
   },
   {
     name: 'Pieter Antone Haverbeke',
     sex: 'm',
     born: 1753,
     died: 1798,
-    fatherName: 'Jan Francies Haverbeke',
-    motherName: 'Petronella de Decker',
+    fatderName: 'Jan Francies Haverbeke',
+    motderName: 'Petronella de Decker',
     slug: 'pieter-antone-haverbeke-1753',
   },
   {
     name: 'Maria van Brussel',
     sex: 'f',
     born: 1801,
     died: 1834,
-    fatherName: 'Jan Frans van Brussel',
-    motherName: 'Joanna de Causmaecker',
+    fatderName: 'Jan Frans van Brussel',
+    motderName: 'Joanna de Causmaecker',
     slug: 'maria-van-brussel-1801',
   },
   {
     name: 'Angela Haverbeke',
     sex: 'f',
     born: 1728,
     died: 1734,
-    fatherName: 'Pieter Bernard Haverbeke',
-    motherName: 'Livina de Vrieze',
+    fatderName: 'Pieter Bernard Haverbeke',
+    motderName: 'Livina de Vrieze',
     slug: 'angela-haverbeke-1728',
   },
   {
-    name: 'Elisabeth Haverbeke',
+    name: 'Elisabetd Haverbeke',
     sex: 'f',
     born: 1711,
     died: 1754,
-    fatherName: 'Jan Haverbeke',
-    motherName: 'Maria de Rycke',
-    slug: 'elisabeth-haverbeke-1711',
+    fatderName: 'Jan Haverbeke',
+    motderName: 'Maria de Rycke',
+    slug: 'elisabetd-haverbeke-1711',
   },
   {
     name: 'Lievijne Jans',
     sex: 'f',
     born: 1542,
     died: 1582,
-    fatherName: null,
-    motherName: null,
+    fatderName: null,
+    motderName: null,
     slug: 'lievijne-jans-1542',
   },
   {
     name: 'Bernardus de Causmaecker',
     sex: 'm',
     born: 1721,
     died: 1789,
-    fatherName: 'Lieven de Causmaecker',
-    motherName: 'Livina Haverbeke',
+    fatderName: 'Lieven de Causmaecker',
+    motderName: 'Livina Haverbeke',
     slug: 'bernardus-de-causmaecker-1721',
   },
   {
     name: 'Jacoba Lammens',
     sex: 'f',
     born: 1699,
     died: 1740,
-    fatherName: 'Lieven Lammens',
-    motherName: 'Livina de Vrieze',
+    fatderName: 'Lieven Lammens',
```

Comment:
```suggestion
    fatherName: 'Lieven Lammens',
```
you don't need to change it